### PR TITLE
Fix SAT warnings

### PR DESCRIPTION
**Pull request by holgerfriedrich**
Reduce number of warnings.

- PMD.SimplifyBooleanReturns where reasonable, suppress where readability is better without change
- PMD.SimplifyBooleanExpressions
- checkstyle.OneStatementPerLineCheck
- checkstyle.NeedBracesCheck
- PMD.UseStandardCharsets
- PMD.UseCollectionIsEmpty
- PMD.UnusedLocalVariable
- checkstyle.ModifierOrderCheck

### CODE REVIEW

**Review by RSStephens — APPROVED** [review]
Lgtm

**Review by theater — APPROVED** [review]

**Review by andrewfg — APPROVED** [review]
Gardena LGTM

**Review by druciak — APPROVED** [review]
Satel LGTM

**Review by GerdZanker — APPROVED** [review]
boschshc LGTM

**Review by hypetsch — APPROVED** [review]
bsblan LGTM

**Review by ccutrer — APPROVED** [review]
homekit LGTM

**Review by Markinus — APPROVED** [review]
km200 LGTM

File: bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/Util.java [hunk 1]
```diff
@@ -33,13 +33,7 @@ public class Util {
...
      */
     public static boolean inRange(int value, int min, int max) {
-        if (value < min) {
-            return false;
-        }
-        if (value > max) {
-            return false;
-        }
-        return true;
+        return !((value < min) || (value > max));
```
Comment (jlaur) [inline bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/Util.java:36]: Consider if:
```suggestion
        return value >= min && value <= max;
```

would be more readable.

File: bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_11/A5_11_03.java [hunk 2]
```diff
@@ -35,6 +35,7 @@ public A5_11_03(ERP1Message packet) {
         super(packet);
     }
 
+    @SuppressWarnings("PMD.SimplifyBooleanReturns")
```
Comment (jlaur) [inline bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/eep/A5_11/A5_11_03.java:38]: Wouldn't this be quite straight-forward here?
```java
// TODO: display error state on thing
return state != 0;
```

**Review by jlaur — CHANGES_REQUESTED** [review]
LGTM - thanks. Two minor comments to consider.

**Review by Wolfgang1966 — APPROVED** [review]
Approved for tacmi Binding. Thank you

**Review by jlaur — APPROVED** [review]
LGTM

**Review by rmichalak — APPROVED** [review]
digiplex LGTM

**Pull request merged**